sphinx_scality/static/css/current.css_t

```diff
@@ -233,6 +233,19 @@ div.body {
     margin: 40px 20px;
 }
 
+/* LINKS */
+
+a:visited {
```

Comment:
If those rules really need to apply to all links in the HTML, then I guess some others could be cleaned-up (at least I remember adding a few `text-decoration: none` in other places).